src/file.ts

```diff
@@ -544,10 +555,55 @@ export class File {
           if (isEmptyDir) {
             await fsp.rmdir(dirPath);
           }
-          return;
         } catch (error) { return; }
       });
-      return fsp.rmdir(dirPath);
+      // }
+      await fsp.rmdir(dirPath);
+    }
+  }
+
+   /**
+    * Recursively copy the folder and contents.
```

Comment:
Would be worth adding an example of using `copyRecursively ` in the common examples (or on README usage).